docs/source/conf.py

```diff
@@ -31,6 +31,9 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    # run sphinx-apidocs on every build
+    'sphinxcontrib.apidoc',
```

Comment:
::question/suggestion:** I have recently switched from `apidoc` to [`autoapi`](https://sphinx-autoapi.readthedocs.io/en/latest/tutorials.html), which I found to be even less intrusive. Have you used it in the past? And if so, are there reasons not to switch?

Reply:
Nope, nothing holding back doing so. Just my non-familiarity with this ecosystem. I migrated in [eb760d8](https://github.com/hermes-hmc/workflow/pull/48/commits/eb760d83d8dbce1b69c88d7e61ed86aa83121d2b)

Please take a look at the RTD preview if this looks good to you.

Reply:
Yes, looking good, feel free to merge.